### DISCUSSION TITLE
Translate the Hook Migration documentation to English. 将 Hook 迁移文档翻译成英文

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#213] [MMEngine]
# Translate the Hook Migration documentation to English. 
 将 Hook 迁移文档翻译成英文 

需要翻译的文档：[https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/migration/hook.md](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/migration/hook.md) 

## Task Information

**Repository**：MMEngine

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：SheffieldCao

**Claim Time**：2023年04月07日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1054

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by chaos1203** [comment]
Claim the mission

**Reply by crazysteeaam** [reply]
The two tasks you have just claimed have been both done in June this year. Please check the label of the discussion, only discussions with StageAwaitAssignment can be claimed~
